roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -779,12 +779,12 @@ impl TweezerDevice {
                 let vec_right = &row[i + 1..].to_vec();
 
                 // Insert the left and right side
-                allowed_shifts.insert(*mid, vec![]);
+                allowed_shifts.entry(*mid).or_default();
                 if let Some(val) = allowed_shifts.get_mut(mid) {
```

Comment:
`or_default()` already gives you a mutable reference to the entry, so I think this line can be removed altogether (but the output of `or_default()` needs to be assigned to a variable `val`

Reply:
makes sense, thanks!